taurus/entity/util.py

```diff
@@ -84,9 +84,9 @@ def array_like():
         try:
             import pandas as pd
             _array_like = (list, tuple, np.ndarray, pd.core.base.PandasObject)
-        except ImportError:
-            _array_like = (list, tuple, np.ndarray)
-    except ImportError:
-        _array_like = (list, tuple)
+        except ImportError:  # pragma: no cover
```

Comment:
This is hard to test because it only occurs if the local environment doesn't have numpy and/or pandas installed.